components/accept-financial-contributions/StartAcceptingFinancialContributionsPage.tsx

```diff
@@ -0,0 +1,582 @@
+            onHostApplyClick={props.onHostApplyClick}
+          />
+        </Box>
+      )}
+    </React.Fragment>
+  );
+}
+
+const I18nMessages = defineMessages({
+  ALL_COUNTRIES: {
+    defaultMessage: 'All countries',
+  },
+});
+
+export default function StartAcceptingFinancialContributionsPage(props: StartAcceptingFinancialContributionsPageProps) {
+  const intl = useIntl();
+  const allCountriesSelectOption = { value: 'ALL', label: intl.formatMessage(I18nMessages.ALL_COUNTRIES) };
+  const [selectedCommunityType, setSelectedCommunityType] = React.useState<string[]>([]);
+  const [selectedCountry, setSelectedCountry] = React.useState(allCountriesSelectOption);
+  const currencyOptions = Currency.map(c => ({ value: c, label: c }));
+  const [selectedCurrency, setSelectedCurrency] = React.useState(currencyOptions[0]);
+  const [searchingForFiscalHost, setSearchingForFiscalHost] = React.useState(false);
+  const router = useRouter();
+  const independentCollectiveOption = router.query.independentCollectiveOption === 'true';
+
+  const countrySelectOptions = React.useMemo(() => {
+    const countryList = isoCountryList
+      .map(countryCode => ({
+        value: countryCode,
+        label: getCountryDisplayName(intl, countryCode),
+      }))
+      .sort((a, b) => a.label.localeCompare(b.label));
+
+    return [allCountriesSelectOption, ...countryList];
+  }, [intl]);
+
+  return (
+    <Flex my={5} alignItems="center" flexDirection="column">
+      <Illustration
+        alt="A place to grow and thrive illustration"
+        src="/static/images/fiscal-hosting/what-is-a-fiscalhost-illustration.png"
+      />
+      <Box mt={4} mb={4}>
+        <H1
+          fontSize={['20px', '32px']}
+          lineHeight={['24px', '40px']}
+          fontWeight="700"
+          color="black.900"
+          textAlign="center"
+        >
+          <FormattedMessage id="contributions.startAccepting" defaultMessage="Start accepting contributions" />
+        </H1>
+        <P mt={3} fontSize="16px" lineHeight="24px" fontWeight="500" color="black.700" textAlign="center">
+          <FormattedMessage defaultMessage="Choose who will hold the money on your behalf" />
+        </P>
+      </Box>
+      <StyledCard width={['300px', '400px', '600px', '800px']} padding="32px 24px">
+        <P mb={1} fontSize="16px" lineHeight="24px" fontWeight="700" color="black.800">
+          <FormattedMessage defaultMessage="What type of community are you?" />
+        </P>
+        <StyledFilters
+          multiSelect
+          filters={CommunityTypes}
+          onChange={setSelectedCommunityType}
+          selected={selectedCommunityType}
+          flexWrap="wrap"
+        />
+
+        <Flex gap="40px">
+          <Box>
+            <P mt={4} fontSize="16px" lineHeight="24px" fontWeight="700" color="black.800">
+              <FormattedMessage defaultMessage="Where would your collective be most active?" />
+            </P>
+            <StyledSelect
```

Comment:
Maybe use https://styleguide.opencollective.com/?path=/story/design-system-inputtypecountry--default-story to make it look nicer with the flags. Unless of course the design dictates specifically not so show the flags. 🤔